hack/deploy-kubevirt.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Since we are going to clone kubemacpool can we use the deploy-kubevirt.sh from kubemacpool ? in fact I think we can even use automation from kubemacpool instead of rewritting stuff here.

Reply:
Currently we dont have it on kubemacpool, should I move it to there?